Scripts/WikiUpdater.cs

```diff
@@ -184,6 +386,13 @@ private List<Wiki.Page.Section> GetMainBodySections(IHtmlElement body)
                         .Select(l => $"[indent]—   {ConvertParagraphToBbcode(l)}[/indent]")
                         .Aggregate((a, b) => a + "\n" + b) + "\n\n";
                     break;
+                case "H3":
```

Comment:
If new tags are being parsed, you'll need to update the page styleguide. I suppose it might need a more general update for everything.

https://wiki.revolutionarygamesstudio.com/wiki/Thriveopedia